src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -201,6 +214,26 @@ private static String getUserId(String userName) {
     }
   }
 
+  private static HttpHost getBorderProxy() {
+
+    EnvType envType = Configuration.getEnvType(Configuration.getEnv());
+    File configFile = new File(Configuration.getCredentialsFilePath());
+
+    switch (envType) {
+      case DEV : {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested code block into a method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)